src/fmu/dataio/tools/rms/volumetrics.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Final
+
+import pandas as pd
+from packaging.version import parse as versionparse
+
+import fmu.dataio as dio
+from fmu.dataio._logging import null_logger
+
+try:
+    import rmsapi
+    from _rmsapi import Project as RmsProject
+except ImportError:
+    pass
+
+try:
+    import roxar as rmsapi
+    from _roxar import Project as RmsProject
+except ImportError:
+    pass
+
+_logger: Final = null_logger(__name__)
+
+# rename columns to FMU standard
+RENAME: Final = {
```

Comment:
Suggest different name than `RENAME` as this is technically a list of curated/allowed column names, and it will definitely be used for some sort of validation at some point. Then this name will be weird, I think.

Reply:
Would it make sense to expand it to allow for differences across RMS versions etc etc, i.e. something like:

```python
RMS_VOLUME_COLUMNS: Final = {  # not great name either
"REAL": {
    "rms": {
        "13.0.3": {
            "name": "Proj. real."
              },
        "default": {
            "name": "Proj. real."
              },
        },
"ZONE": {
    ...
    },
...
}

Reply:
I will take a look into it

Reply:
Until there is a concrete need to split it on RMS versions, we can do that. For now I make a private dictionary

Reply:
When such a concrete need arise, will it happen fast or slow? If it will happen fast, it might be worth preparing up front. If we expect to have (enough) time (multiple months. People availability & capacity, Komodo release cycle, etc), I agree that it can wait.